rust/sbp/Cargo.toml

```diff
@@ -33,6 +33,11 @@ log = "0.4"
 nom = "6.0"
 thiserror = "1.0"
 
+[dependencies.swiftnav-rs]
+git = "https://github.com/swift-nav/swiftnav-rs"
+tag = "v0.4.0"
+optional = true
```

Comment:
Hmm not sure if that's a problem. I think as long as this crate doesn't enable that feature in swiftnav-rs we should be alright? 

Either way, I think `SBPMessage` should have a method like this, and this is just the first way that popped into my head on how to do it. Definitely would be open to other ideas. There are three apps that I can think of that read messages and at some point want to check if the current message is time aware - console, choppa, and icbins. So ideally I think this functionality should be provided from one crate

Reply:
We should move the libsbp->swiftnav-rs structs feature into libsbp then?

Reply:
I'm cool with moving it, but it would be best to make sure these integration features all live in one spot. Whether that's in libsbp or swiftnav-rs doesn't really matter much to me.

Reply:
👍